HM3CoreKill/HM3CoreKill/ck/HM3InGameTools.cpp

```diff
@@ -129,7 +129,7 @@ namespace ck
 				ImGui::Text("Noise level: "); ImGui::SameLine(0.f, 15.f);
 
 				ImVec4 noiseLevelColor = ImVec4(0.f, 1.f, 0.f, 1.f);
-				if (osd->m_realNosieLevel >= 0.f && osd->m_realNosieLevel <= 40.f)
```

Comment:
Left part of condition (`osd->m_realNosieLevel >= 0.f`) should be changed too.
Also as other cases further.